flow-typed/global.js

```diff
@@ -0,0 +1 @@
+declare var PaymentRequest: any;
```

Comment:
Does flow throw errors without this?

Reply:
Yes, "Could not resolve name PaymentRequest"

Reply:
I avoid that by using `window.PaymentRequest` ec2788e